splinterd/api/static/openapi.yaml

```diff
@@ -2534,6 +2592,18 @@ components:
           description: "The profile picture for the user's account"
           example: "https://lh6.googleusercontent.com/AMZpcempiaceeiA/s96-c/photo.jpg"
 
+    OAuthUser:
+      type: object
+      properties:
+        subject:
+          type: string
+          description: "Oauth provider's unique identifier for the user"
+          example: "129367690558975733497"
+        user_id:
+          type: string
+          description: "Internal unique identifier for the user"
```

Comment:
Instead of "Internal" should this say `Biome`?

Reply:
This terminology is used throughout the Biome endpoints, which return the equivalent Biome user_id. But, perhaps it would make sense in this instance to add `Biome` to the description since the ID isn't as obviously related to Biome in this endpoint. updating ...

Reply:
Updated to say `Biome's internal unique identifier for the user`